.vscode/launch.template.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "type": "node",
+        "name": "Attach",
+        "port": 9229,
+        "request": "attach",
+        "skipFiles": ["<node_internals>/**"]
+      },
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "Beacon",
+        "skipFiles": ["<node_internals>/**"],
+        "smartStep": true,
+        "program": "${workspaceFolder}/packages/cli/bin/lodestar.js",
+        "args": ["beacon"],
+        "console": "integratedTerminal"
+      },
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "Dev",
+        "skipFiles": ["<node_internals>/**"],
+        "smartStep": true,
+        "program": "${workspaceFolder}/packages/cli/bin/lodestar.js",
+        "args": ["dev"],
+        "console": "integratedTerminal"
+      },
+      {
+        "name": "Test Current File",
+        "type": "node",
+        "request": "launch",
+        "program": "${workspaceFolder}/node_modules/.bin/vitest",
+        "args": ["--run", "${file}", "-t", "${input:testName}"],
+        "cwd": "${workspaceFolder}/${input:packageName}",
+        "console": "integratedTerminal",
+        "skipFiles": ["<node_internals>/**"]
+      }
+    ],
+    "inputs": [
+      {
```

Comment:
Would it be possible to do another input block to extract the test type so it applies the correct vitest config?  

```json
        "id": "testType",
        "type": "command",
        "command": "extension.commandvariable.transform",
        "args": {
          "text": "${relativeFileDirname}",
          "find": "^(test/[^/]+).*",
          "replace": "$1"
        }
```

That way in the test config we could do something like this
```json

            "program": "${workspaceFolder}/node_modules/.bin/vitest",
            "args": [
                "--run",
                "--pool",
                "threads",
                "--poolOptions.threads.singleThread",
                "--config",
                "vitest.${input:testType}.config.ts",
                "-t",
                "${input:testName}",
                "${file}"
            ],
```